lib/Backend/GlobOpt.cpp

```diff
@@ -13931,21 +13931,14 @@ GlobOpt::CheckJsArrayKills(IR::Instr *const instr)
             break;
         }            
 
-        case Js::OpCode::InitClass:
+        case Js::OpCode::NewClassProto:
             Assert(instr->GetSrc1());
-            if (instr->GetSrc2() == nullptr)
+            if (instr->GetSrc1()->IsAddrOpnd() && instr->GetSrc1()->AsAddrOpnd()->m_address == (Js::Var)func->GetScriptContextInfo()->GetObjectPrototypeAddr())
```

Comment:
Since this test is repeated, do you think it's worth making a function for it (e.g. `IsObjectPrototypeAddr` or similar)?

Reply:
Yeah, not a bad idea. Or a test that the constant embedded in an AddrOpnd is equal to some given address.